bin/reth/src/commands/node/events.rs

```diff
@@ -158,6 +158,21 @@ impl<DB> NodeState<DB> {
                     self.current_stage = None;
                 }
             }
+            PipelineEvent::Unwind { stage_id, input } => {
+                let current_stage = CurrentStage {
+                    stage_id,
+                    eta: match &self.current_stage {
+                        Some(current_stage) if current_stage.stage_id == stage_id => {
+                            current_stage.eta
+                        }
```

Comment:
I think this is incorrect because it might be the ETA from execution, and not unwinding

Reply:
@shekhirin any pointer on how i could calculate the ETA on unwinding?

Reply:
sorry for the really delayed response

to unblock this i'd just always return `Eta::default` in this case. the `Eta` struct is not really built for time estimation in reverse

we can open up a follow up issue on making eta's work in reverse if we need it (+ linking this pr in that issue), but most of the time you won't unwind a lot so it might not matter too much.